### PR TITLE
[categories] Updated translations

### PR DESCRIPTION
**Pull request by liondog**
Updated some translations and added translations for railway-level_crossing (OSM equivalent: railway=level_crossing).

### CODE REVIEW

**Comment by mpimenov** [comment]
Please sign the CLA: https://maps.me/cla/

**Comment by liondog** [comment]
I've already signed it.

File: data/categories.txt [hunk 1]
```diff
@@ -1807,6 +1807,21 @@ el:^Υπόγειος|σωλήνας|μετρό|υπόγεια|μεταφορές
...
+de:^Bahnübergang
+it:^Passaggio a livello
+es:^Paso a nivel
+pt:^Passagem de nível
+pl:^Przejazd kolejowy
+cs:^Železniční přejezd
+nl:^Overweg
+da:^Jernbaneoverskæring
+fi:^Tasoristeys
+hu:^Útátjáró
```
Comment (mpimenov) [inline data/categories.txt]: What about other languages?

**Comment by mpimenov** [comment]
@Zverik PTAL

File: data/categories.txt (hunk 1 above)
Comment (liondog) [reply data/categories.txt]: What are your contribution guidelines regarding translations? What languages are at least necessary?

Comment (mpimenov) [reply data/categories.txt]: The only necessary language is English, but we try to add as many as possible at least in the simple situations such as categories.txt where Grammar is not needed.
Here is the list: https://github.com/mapsme/omim/blob/master/indexer/categories_holder.cpp#L152

**Comment by mpimenov** [comment]
The merge was unintended.
We've discussed this offline and won't revert the changes but please consider adding all languages if you plan to add strings in the future, at least for simple words like "Basketball".

**Pull request merged**